test/demo/customheaders.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```

Comment:
Great to have this demo! I haven't run it yet... does it work on a regular local grunt server?